app/services/api/v3/npq_applications_query.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Api
+  module V3
+    class NPQApplicationsQuery
+      INITIAL_COHORT_YEAR = 2021
```

Comment:
suggestion: maybe we can use 2020 and more than similar to other queries, so when we get to clean this and maybe add a scope it's easier to find in the code

Reply:
Yeah that's a sensible idea I'll update it to be consistent 👍